ecommers/lib/ui/widgets/category/categories_compact_view.dart

```diff
@@ -8,8 +8,7 @@ import 'package:ecommers/ui/widgets/category/category_item.dart';
 import 'package:flutter/material.dart';
 
 class CategoriesCompactView extends StatelessWidget {
-  static const _containerHeight = 134.0;
-  static const categoryItemSize = Size(74.0, 89.0);
+  static const categoryItemSize = CategoryItem.size;
```

Comment:
can you remove categoryItemSize ? and use in code CategoryItem.Size

Reply:
done